mtags/src/main/scala-3/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -39,30 +38,24 @@ class CompletionProvider(
   }
 
   extension (l: List[Completion]) {
-    def filterInteresting: List[Completion] = {
+    def filterInteresting: List[Completion] =
       val isSeen = mutable.Set.empty[String]
       val buf = List.newBuilder[Completion]
-      def isInteresting(head: Completion): Boolean = {
+      def isInteresting(head: Completion): Boolean =
         val id = head.sym.show
         def isNotLocalForwardReference: Boolean =
           !head.sym.isLocalToBlock ||
             !head.sym.srcPos.isAfter(pos) ||
             head.sym.is(Param)
 
-        if (
-          !isSeen(id) &&
-          !isUninterestingSymbol(head.sym) &&
-          isNotLocalForwardReference
-        ) {
+        if !isSeen(id) &&
+        !isUninterestingSymbol(head.sym) &&
+        isNotLocalForwardReference then
```

Comment:
It seems like the indentation is missing  one second and third line
```scala
if !isSeen(id) &&
   !isUninterestingSymbol(head.sym) &&
   isNotLocalForwardReference then
```

Reply:
actually, it is needed, @dos65 is right. (not because of significant indentation, of course, but because multiline infix needs to be indented if not within `{}`.) the only question is: should we break before `then`? i.e.

```
if !isSeen(id) &&
  !isUninterestingSymbol(head.sym) &&
  isNotLocalForwardReference then
   isSeen += id
...
```
or
```
if !isSeen(id) &&
  !isUninterestingSymbol(head.sym) &&
  isNotLocalForwardReference
then
   isSeen += id
```
i find the second one more readable. @tgodzik @dos65 what do you think?

Reply:
Definitely, the second looks more readable. 🤔

Reply:
is it valid scala3? :) before i implement this rule...

Reply:
@kitbellew yep, it's valid )

Reply:
scalameta/scalafmt#2491